pipelines/tekton/azureml-container-pipeline/model-upload/local-model-to-pvc-pod.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: local-model-to-pvc-pod-template
+objects:
+- apiVersion: v1
+  kind: Pod
+  metadata:
+    name: local-model-to-pvc-pod
+  spec:
+    volumes:
+      - name: ${PVC}
+        persistentVolumeClaim:
+          claimName: ${PVC}
+    containers:
+    - name: local-model-to-pvc-container
+      image: ubi9
+      stdin: true
+      tty: true
+      securityContext:
+        allowPrivilegeEscalation: false
+      volumeMounts:
+          - mountPath: /workspace/${PVC}
+            name: ${PVC}
```

Comment:
I think you could consider putting this into the object list of the other template, if they're always expected to be processed together, and the `PVC` parameter will always have the same value. WDYT?

Reply:
The PVC and the pod want to be deleted at different times, this is because when you upload the model file to the PVC it is mounted to the pod. For another pod to mount the PVC the pod we create must un-mount it as you can't have both mounting at the same time.

So when you are deleting them you want to do it separately.

Reply:
Do we really want to rely on the same PVC being remounted for various uses?

Shouldn't the Pod writing the model to the PV use one PVC, and then Pods consuming the model different PVC(s), possibly with different selectors for labels set by the populating Pod after it has finished storing the model?

Reply:
Yes the pod writing the model does only use one PVC, it is mounted then `oc cp` is used and then unmounted and the pod is deleted.

For our usecase the PVC we upload to is then used as a workspace in the pipelines, so the PVC it uses for the pipelines can copy over the model file. So I think adding labels etc. isn't needed for this. (Atleast I don't think so)